hack/helm/values.yaml

```diff
@@ -0,0 +1,16 @@
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+hncIgnoreNamespaces:
```

Comment:
`hncExcludeNamespaces` would be better, and you can also point to the right part of the [docs](https://github.com/kubernetes-sigs/hierarchical-namespaces/blob/master/docs/user-guide/how-to.md#excluding-namespaces-from-hnc) in a comment (as you can tell, I'm pretty big on comments!).

However, simply adding a namespace here will _not_ actually exclude the namespace; you'd also have to add the `----excluded-namespace` option to the HNC deployment. So maybe it's better to hardcode these for now, unless you want to somehow add it to the Deployment in a template?

Reply:
renamed.

> However, simply adding a namespace here will not actually exclude the namespace

Really? My understanding of the docs was labelling namespaces appropriately would tell the HNC controller to ignore them.

Reply:
No, it's a two-step process.
* Why not just let people add the label to the namespace (after all, this is what Kubernetes itself cares about)? Because if we did, anyone who could edit a namespace could exempt themselves from HNC's oversight simply by adding the label, and we don't want to allow that. As a result, if you try to add this label to any normal namespace, HNC will stop you.
* Why not just add the namespace to the `--exclude-namespace` flag, and ask HNC itself to label the namespace on our behalf? Because we don't want HNC to automatically be making changes to critical namespaces like `kube-system`, and users may not want to give it permission to do so anyway.

As a result, we require _both_. The admin of HNC needs to say `--exclude-namespace`, which (now that I think about it) probably should have been called `--allow-namespace-exclusion`, because it gives us _permission_ to add the label to the namespace. And then, someone with edit access to that namespace needs to actually exclude it. Or, if you haven't installed HNC yet, you can do them in the reverse order.

Does that make sense?